### PR TITLE
[2.19.x] G-9513 allow aliases for sharing groups

### PR DESCRIPTION
**Pull request by andrewzimmer**
#### What does this PR do?
Allows aliases for sharing groups

#### Who is reviewing it? 
@abel-connexta @cassandrabailey293 @hayleynorton @zta6 

#### How should this be tested?
- build /install / run
- verify no regression with sharing modal

#### What are the relevant tickets?
Fixes: #____

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: ui/packages/catalog-ui-search/src/main/webapp/extension-points/sharing-aliases/index.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+type shaingAliasType = {
```
Comment (cassandrabailey293) [inline ui/packages/catalog-ui-search/src/main/webapp/extension-points/sharing-aliases/index.tsx:15]: ✏️  typo - sha**r**ingAliasType

**Comment by rymach** [comment]
what's the benefit of aliases over changing the group names?

**Comment by andrewzimmer** [comment]
@rymach changing the actual group name and all the references in the backend would be complicated and not sure what benefit that would provide. Also not sure if I could make that change in ddf

**Comment by andrewzimmer** [comment]
build now

**Review by cassandrabailey293 — APPROVED** [review]

**Comment by cassandrabailey293** [comment]
hero successful 🎉 
verified no regression in the sharing modal and verified alias downstream

**Comment by shaundmorris** [comment]
Merging. Build failed with known docker issue. Hero was successful.

**Pull request merged**